massa-graph/src/settings.rs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2021 MASSA LABS <info@massa.net>
+
+#![allow(clippy::assertions_on_constants)]
+
+use massa_models::Amount;
+use massa_signature::PrivateKey;
+use massa_time::MassaTime;
+use serde::{Deserialize, Serialize};
+use std::{path::PathBuf, usize};
+
+/// Consensus full configuration (static + user defined)
+///
+/// Assert that `THREAD_COUNT >= 1 || T0.to_millis() >= 1 || T0.to_millis() % THREAD_COUNT == 0`
+#[derive(Debug, Deserialize, Serialize, Clone)]
+pub struct LedgerConfig {
+    /// Number of threads
+    pub thread_count: u8,
```

Comment:
So you duplicate thread count in all the configs that need it right ?

Reply:
The thread count management will be the object of another issue/PR

Reply:
#2197